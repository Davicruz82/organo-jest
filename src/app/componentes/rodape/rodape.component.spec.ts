import { ComponentFixture, TestBed } from "@angular/core/testing";
import { RodapeComponent } from "./rodape.component";

describe('RodapeComponente', () => {
	let component: RodapeComponent;
	let fixture: ComponentFixture<RodapeComponent>;

	beforeEach(() => {
		TestBed.configureTestingModule({
			imports: [RodapeComponent],
		});
		fixture = TestBed.createComponent(RodapeComponent);
		component = fixture.componentInstance;
	});

	it('deveria ser criado', () => {
		expect(component).toBeTruthy();
	});

	it('deveria definir as propriedades alt e src', () => {
		expect(component.alt).toBeDefined();
		expect(component.src).toBeDefined();
	});

	it('deveria renderizar o conteÃºdo na tela baseado nas propriedades alt e src', () => {
		component.src = 'https://example.com/test-image.jpg';
    component.alt = 'Imagem teste';
		expect(component).toMatchSnapshot();
	});
});