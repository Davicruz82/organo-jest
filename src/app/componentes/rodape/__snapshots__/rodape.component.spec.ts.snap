// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RodapeComponente deveria renderizar o conte√∫do na tela baseado nas propriedades alt e src 1`] = `
RodapeComponent {
  "__ngContext__": 4,
  "alt": "Imagem teste",
  "src": "https://example.com/test-image.jpg",
}
`;
